external/js/cothority/src/byzcoin/beautifier/utils.ts

```diff
@@ -0,0 +1,11 @@
+
+export interface IJSONSchema {
+    status: 0 | 1;
+    type: "spawn" | "invoke" | "delete" | "UNKNOWN";
+    contract: string;
+    args: IArgsSchema;
+}
+
+export interface IArgsSchema {
+    [index: number]: {name: string, value: string, full?: string};
```

Comment:
What's the advantage of this against
```
export interface IArgument {
  name: string;
  value: string;
  full?: string;
}
```
And then use it like:
```
static Invoke(args: Argument[]): IArgument[]
```
?

Reply:
Now that I read my own comment I think that `Argument` and `IArgument` are too close. Would `IArgumentDetail` or something be better?